include/api/wasmedge/wasmedge.h

```diff
@@ -2772,6 +2772,14 @@ WasmEdge_ImportObjectCreateWasmEdgeProcess(const char *const *AllowedCmds,
                                            const uint32_t CmdsLen,
                                            const bool AllowAll);
 
+/// Get the module name.
+///
```

Comment:
Please add the comment.

```
/// The returned string object is linked to the module name of the import object context, and the caller should __NOT__ call the `WasmEdge_StringDelete`.
```

Reply:
Accepted. I'll Add the doc comments. Thanks a lot!

Reply:
The issue has been fixed. Thanks!